docs/semgrep-ci/overview.md

```diff
@@ -65,6 +69,22 @@ semgrep-agent --baseline-ref main
 
 To connect your Semgrep CI scans to Semgrep App, you can optionally provide the following environment variables:
 
+#### Sample configurations
+
+These instructions have been used on the following providers by the community:
+
+- Bitbucket Pipelines TODO ask why there are no links
+- Bitrise
+- Buildbot
+- TeamCity CI
+- Codefresh
+- Travis CI
+- Buildkite [(sample configuration)](../sample-ci-configs/#buildkite)
+- Jenkins [(sample configuration)](../sample-ci-configs/#jenkins)
+- CircleCI [(sample configuration)](../sample-ci-configs/#circleci) 
```

Comment:
@pabloest maybe you can help here. 🙏 

There used to be [this table](https://semgrep.dev/docs/semgrep-ci/overview/#run-semgrep_agent) with other CI providers. It included GitHub and GitLab as well, but we already link to these samples of configuration in the sections dedicated to the above. I guess that the document evolved organically and this table might have been one of the older foundations.
 
I made a list from this table as we do not have to make a table with two columns for a list essentially (there are no values different in nature from the first column in the second one).

Action: Please, do we need to have CI providers such as Codefresh or Bitrise listed here if we do not provide sample CI configs for them? If yes, who might provide us with these sample configs for Semgrep CI (we were checking this together with @brendongo so he was not sure as well). And sorry, they are not sorted out alphabetically, I may sort them out.

Reply:
According to the docs it seems at some point one or more users reported success in running Semgrep on those CI platforms. However I haven’t been able to find evidence of that for Codefresh or Bitrise. But I have seen it for Travis CI (for example). So I would say where we can see evidence of success in a CI platform, let’s list it even if we don’t provide a sample config (that would be even better, of course).

Reply:
That is nice @pabloest, adding as the following text:

`The community succesfully run Semgrep CI on the following platforms (some include a link to sample configuration):

- Bitbucket Pipelines TODO ask why there are no links
- Bitrise
- Buildbot
- Buildkite [(sample configuration)](../sample-ci-configs/#buildkite)
- CircleCI [(sample configuration)](../sample-ci-configs/#circleci) 
- Codefresh
- Jenkins [(sample configuration)](../sample-ci-configs/#jenkins)
- TeamCity CI
- Travis CI`

Reply:
Thanks!

I suggest this edit: “…successfully ran Semgrep in the following CI platforms…”

Reply:
@pabloest oh thank you, forgot to use past tense. 🤦